app/controllers/demo_mode/sessions_controller.rb

```diff
@@ -50,11 +50,11 @@ def begin_demo!
 
     def render_signinable_json
       if @session.signinable.blank?
-        render json: { id: @session.id, processing: true }
+        render json: { id: @session.id, processing: @session.processing?, failed: @session.failed? }
```

Comment:
Maybe we could do something like this:

```ruby
render json: { id: @session.id, processing: @session.processing?, status: @session.status }
```

It sounds like we want to keep processing for compatibility reasons, but otherwise, I think a status is the preferred way to represent this.